src/sentry/conf/server.py

```diff
@@ -675,6 +676,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("release_health.duplex", routing_key="release_health.duplex"),
     Queue("get_suspect_resolutions", routing_key="get_suspect_resolutions"),
     Queue("get_suspect_resolutions_releases", routing_key="get_suspect_resolutions_releases"),
+    Queue("replays", routing_key="replays"),
```

Comment:
should we name this to be a little more specific in case we add more celery tasks in the future? e.g. `replays_deletion`

Reply:
what would be our strategy there? do our other celery queues have routing mechanisms within the tasks?